frontend/src/components/Apps/AppModal/AddAppModal.tsx

```diff
@@ -0,0 +1,68 @@
+import styles from './AddAppModal.module.css';
+import { useState } from 'react';
+
+interface ModalProps {
+	onAdd: (a: string, b: string, c: string) => void;
+	onClose: (a: boolean) => void;
+}
+
+const AddAppModal: React.FunctionComponent<ModalProps> = ({
+	onAdd,
+	onClose,
+}: ModalProps) => {
+	const [name, setName] = useState<string>('');
```

Comment:
Could we, instead of a bunch of useStates, have one useReducer for everything state wise?

Reply:
I was deciding between a couple of options, useReducer included, as well as useState as an object containing the info but figured it was more readable through useState on the separate inputs. Will be sure to update it to useReducer in a future update.